cinema/serializers.py

```diff
@@ -44,14 +50,34 @@ class MovieListSerializer(MovieSerializer):
         many=True, read_only=True, slug_field="full_name"
     )
 
+    class Meta:
+        model = Movie
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image",
+        )
+
 
 class MovieDetailSerializer(MovieSerializer):
     genres = GenreSerializer(many=True, read_only=True)
     actors = ActorSerializer(many=True, read_only=True)
 
     class Meta:
```

Comment:
try:
```
class Meta(MovieSerializer.Meta):
    pass
```

Or try without meta

Reply:
Class MovieListSerializer.Meta was redefined in order to add field "image" because task description requires to show one in Movie list endpoint. MovieSerializer.Meta doesn't have "image" in it's fields tuple.